backend/src/BuiltinCliHost/BuiltinCliHost.fsproj

```diff
@@ -19,6 +19,7 @@
     <ProjectReference Include="../BuiltinExecution/BuiltinExecution.fsproj" />
     <ProjectReference Include="../BuiltinCli/BuiltinCli.fsproj" />
     <ProjectReference Include="../LibCliExecution/LibCliExecution.fsproj" />
+    <ProjectReference Include="../LibCloud/LibCloud.fsproj" />
```

Comment:
will revert this and related changes - locally I use the non-web-based one because the `dark-packages`-accessing one is too slow
.. unless you'd like that as well, for now?